src/ContentHubClient.php

```diff
@@ -26,7 +29,10 @@
  *
  * @package Acquia\ContentHubClient
  */
-class ContentHubClient extends Client {
+class ContentHubClient implements ClientInterface {
+
+  use ClientTrait;
```

Comment:
ClientTrait is not available in guzzle <7.0, however, Drupal 9.x supports guzzle 6. In this particular case the best would be to actually just implement the methods, get, put, delete, post and use the client we already have to execute those requests.